test/Actions.test.ts

```diff
@@ -130,4 +130,210 @@ contract('Actions', ([owner, random]) => {
       assert.equal(depositArgs.index, new BN(index))
     })
   })
+
+  describe('Parse Mint Arguments', () => {
+    it('should not be able to parse a non Mint action', async () => {
+      const actionType = ActionType.OpenVault
+      const asset = ZERO_ADDR
+      const vaultId = '0'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: owner,
+        sender: owner,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(actionTester.testParseMintAction(data), 'Actions: can only parse arguments for mint actions')
+    })
+    it('should not be able to parse an invalid sender address', async () => {
+      const actionType = ActionType.MintShortOption
+      const asset = ZERO_ADDR
+      const vaultId = '0'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: ZERO_ADDR,
+        sender: owner,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(actionTester.testParseMintAction(data), 'Actions: cannot mint from an invalid account')
+    })
+    it('should be able to parse arguments for a mint long action', async () => {
+      const actionType = ActionType.MintShortOption
+      const asset = ZERO_ADDR
+      const vaultId = '1'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: owner,
+        sender: random,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await actionTester.testParseMintAction(data)
+
+      const mintArgs = await actionTester.getMintArgs()
+      assert.equal(mintArgs.owner, owner)
+      assert.equal(mintArgs.amount, new BN(amount))
+      assert.equal(mintArgs.otoken, asset)
+      assert.equal(mintArgs.to, random)
+      assert.equal(mintArgs.vaultId, new BN(vaultId))
+      assert.equal(mintArgs.index, new BN(index))
+    })
+    it('should be able to parse arguments for a mint collateral action', async () => {
```

Comment:
It shouldn't be a `mint collateral` test right? Not sure what this test is for, it seems similar to the previous one.